src/components/interfaces/HMI_API.xml

```diff
@@ -1887,19 +1990,25 @@
    </param>
    <param name="ventilationMode" type="Common.VentilationMode" mandatory="false">
    </param>
+   <param name="heatedSteeringWheelEnable" type="Boolean" mandatory="false">
+       <description>value false means disabled/turn off, value true means enabled/turn on.</description>
+   </param>
+   <param name="heatedWindshieldEnable" type="Boolean" mandatory="false">
+       <description>value false means disabled, value true means enabled.</description>
+   </param>
+   <param name="heatedRearWindowEnable" type="Boolean" mandatory="false">
+       <description>value false means disabled, value true means enabled.</description>
+   </param>
+   <param name="heatedMirrorsEnable" type="Boolean" mandatory="false">
+       <description>value false means disabled, value true means enabled.</description>
+   </param>
  </struct>
 
    <struct name="ClimateControlCapabilities">
    <description>Contains information about a climate control module's capabilities.</description>
    <param name="moduleName" type="String" maxlength="100" mandatory="true" >
      <description>The short name or a short description of the climate control module.</description>
    </param>
-   <param name="currentTemperatureAvailable" type="Boolean" mandatory="false">
-     <description>
-       Availability of the reading of current temperature.
-       True: Available, False: Not Available, Not present: Not Available.
-     </description>
-   </param>
```

Comment:
If it already exists in the HMI API, it shouldn't be removed from here. This discrepancy should be handled in another PR.